src/calls.rs

```diff
@@ -14,7 +19,7 @@ use crate::{
 ///   // do awesome stuff
 /// }
 /// ```
-#[derive(Debug, Clone)]
+#[derive(Default, Clone, Serialize, Deserialize, Debug)]
```

Comment:
I'm not really sure I understand the need for this `Calls` object over just a normal `Vec<Call>`?